src/Raven.Client/Documents/Session/DocumentSessionRevisions.cs

```diff
@@ -122,6 +129,7 @@ public long GetCountFor(string id)
         {
             var operation = new GetRevisionsCountOperation(id);
             var command = operation.CreateRequest();
+            if (command == null) return command.Result;
```

Comment:
I'm looking into it and I wonder why I changed it. It doesn't make any sense to me whatsoever.
there's no need to caching over here because it's returning the number of revisions and that's it.
the only thing I will change here is the IncrementNumberOfRequests. sounds good?